resources/functionMap.php

```diff
@@ -13272,7 +13272,7 @@
 'xdebug_get_declared_vars' => ['array'],
 'xdebug_get_formatted_function_stack' => [''],
 'xdebug_get_function_count' => ['int'],
-'xdebug_get_function_stack' => ['array', 'message='=>'string', 'options='=>'int'],
+'xdebug_get_function_stack' => ['array', 'options='=>'array'],
```

Comment:
Would be nice to introduce array shape with the actual options.

Reply:
Added